scripts/common_click.inc

```diff
@@ -252,11 +252,12 @@ function drawWater()
     local water = srFindImage("iconWaterJugSmall.png", 1);
     if water then
       safeClick(water[0]+3, water[1]-5);
+	  lsSleep(click_delay*5);
       local max = waitForImage("crem-max.png", 500, "Waiting for Max button");

```

Comment:
The problem was that it would detect the button and then somehow not manage to click it, but the added wait made it succeed a lot more often.